packages/assets/src/loader/parsers/textures/loadVideo.ts

```diff
@@ -0,0 +1,58 @@
+import { BaseTexture, extensions, ExtensionType, utils, VideoResource } from '@pixi/core';
+import { checkDataUrl } from '../../../utils/checkDataUrl';
+import { checkExtension } from '../../../utils/checkExtension';
+import { LoaderParserPriority } from '../LoaderParser';
+import { createTexture } from './utils/createTexture';
+
+import type { IBaseTextureOptions, Texture } from '@pixi/core';
+import type { Loader } from '../../Loader';
+import type { LoadAsset } from '../../types';
+import type { LoaderParser } from '../LoaderParser';
+
+const validVideoExtensions = ['.mp4', '.webm', '.ogv'];
```

Comment:
One thing I concern is that .ogg can be audio or video, which may conflict with [@pixi/sound](https://github.com/pixijs/sound).

Reply:
`.mp4` could be audio-only too.

Reply:
lets make an assumption that:

- .mp4 is video 
- .ogg is sound

We recently updated the Assets package so that the parser can easily be overridden.  In that edge case where an ogg is a video or mp4 is a sound, users can specify this themselves.